var/spack/repos/builtin/packages/openjdk/package.py

```diff
@@ -18,6 +18,12 @@
 #    format returned by platform.system() and 'arch' by platform.machine()
 
 _versions = {
+    "21_35": {
```

Comment:
Based on https://adoptium.net/temurin/archive/, there are also 21.0.3+9, so it will make sense to allow for this by renaming:
```suggestion
    "21.0.0_35": {
```
At some later point someone can then add 21.0.3_9, and it would get sorted correctly in regards to 21.0.0_35 (but maybe not in regards to 21_35).

Reply:
good catch. thanks